react/components/RadioGroup/index.js

```diff
@@ -10,48 +10,75 @@ class RadioGroup extends React.Component {
   }
 
   render() {
-    const { options, value, name, disabled, hideBorder } = this.props
+    const {
+      options,
+      value,
+      name,
+      disabled,
+      hideBorder,
+      label,
+      testId,
+      size,
+    } = this.props
+
+    const large = size === 'large'
 
     return (
       <div>
-        {options.map((option, i) => {
-          const isFirst = i === 0
-          const isLast = i === options.length - 1
-          const isDisabled = disabled || option.disabled
-          const id = `${name}-${i}`
-          return (
-            <label
-              className={`db br3 ${classNames({
-                'b--muted-4 ba pv2 ph4': !hideBorder,
-                pointer: !isDisabled,
-              })}`}
-              key={id}
-              style={{
-                ...(!isFirst && {
-                  borderTopLeftRadius: 0,
-                  borderTopRightRadius: 0,
-                  borderTop: 'none',
-                }),
-                ...(!isLast && {
-                  borderBottomLeftRadius: 0,
-                  borderBottomRightRadius: 0,
-                }),
-              }}>
-              <div className={classNames({ mt3: !hideBorder })}>
-                <Radio
-                  id={id}
-                  isLast={isLast}
-                  name={name}
-                  disabled={isDisabled}
-                  onChange={this.handleChange}
-                  label={option.label}
-                  value={option.value}
-                  checked={value === option.value}
-                />
-              </div>
-            </label>
-          )
-        })}
+        <fieldset
+          className="vtex-input w-100"
+          data-testid={testId}
+          style={{ border: 'none', padding: 0 }}>
+          <legend>
```

Comment:
Should the `<legend>` element exist if there's no `label`?

Reply:
Fixed it!